components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -14,6 +14,7 @@
   export let access_token: string = "";
   export let event_title: string;
   export let event_description: string;
+  export let show_hosts: "show" | "hide";
```

Comment:
Just curious about this choice because it seems like it could be a boolean

Reply:
Agreed, but radio buttons can't have booleans as values (only strings, numbers)

Reply:
right, right. `!!` could be used, but I think it'd make it less readable